test/MultiCompiler.test.js

```diff
@@ -299,9 +299,13 @@ describe("MultiCompiler", function() {
 			beforeEach(function() {
 				setupTwoCompilerEnvironment(env);
 				env.callback = sinon.spy();
-				env.options = {
-					testWatchOptions: true
-				};
+				env.options = [{
```

Comment:
Fixing these, but FYI the previous format was suggested by the linter, not sure why